optax/_src/loss.py

```diff
@@ -454,3 +454,49 @@ def ctc_loss(logits: chex.Array,
       logits, logit_paddings, labels, label_paddings,
       blank_id=blank_id, log_epsilon=log_epsilon)
   return per_seq_loss
+
+
+def kl_divergence(log_predictions: chex.Array,
+                targets: chex.Array) -> chex.Array:
```

Comment:
The second line here isn't correctly aligned now (but I can also fix that during merging).

Reply:
done!